--- cmake/SndFileChecks.cmake.orig	2022-03-27 13:48:57.000000000 +0200
+++ cmake/SndFileChecks.cmake	2022-04-27 09:31:10.000000000 +0200
@@ -23,12 +23,6 @@ else ()
 endif ()
 set (SF_COUNT_MAX 0x7fffffffffffffffll)
 
-if (CMAKE_SYSTEM_NAME STREQUAL "OpenBSD")
-	find_package (Sndio)
-elseif (NOT WIN32)
-	find_package (ALSA)
-endif ()
-
 if (VCPKG_TOOLCHAIN AND (NOT CMAKE_VERSION VERSION_LESS 3.15))
 	set (CMAKE_FIND_PACKAGE_PREFER_CONFIG ON)
 endif ()
