--- cmake/SndFileChecks.cmake.orig	2022-12-25 11:19:16
+++ cmake/SndFileChecks.cmake	2023-01-10 16:04:33
@@ -16,12 +16,6 @@ endif ()
 	add_definitions(${LARGE_FILES_DEFINITIONS})
 endif ()
 
-if (CMAKE_SYSTEM_NAME STREQUAL "OpenBSD")
-	find_package (Sndio)
-elseif (NOT WIN32)
-	find_package (ALSA)
-endif ()
-
 if (VCPKG_TOOLCHAIN AND (NOT CMAKE_VERSION VERSION_LESS 3.15))
 	set (CMAKE_FIND_PACKAGE_PREFER_CONFIG ON)
 endif ()
@@ -49,16 +43,11 @@ endif ()
 	set (HAVE_EXTERNAL_XIPH_LIBS 0)
 endif ()
 
-find_package (mp3lame)
-find_package (Mpg123 1.25.10)
 if (TARGET mp3lame::mp3lame AND (TARGET MPG123::libmpg123))
 	set (HAVE_MPEG_LIBS 1)
 else ()
 	set (HAVE_MPEG_LIBS 0)
 endif()
-
-find_package (Speex)
-find_package (SQLite3)
 
 check_include_file (byteswap.h		HAVE_BYTESWAP_H)
 check_include_file (dlfcn.h			HAVE_DLFCN_H)
